pages/k8s/metallb.md

```diff
@@ -42,13 +42,45 @@ explanation of the required configuration</a> from the
 
 </p></div>
 
-Currently, the best way to install MetalLB on Charmed Kubernetes is with a helm chart:
+The best way to install MetalLB in layer 2 mode on Charmed Kubernetes is with 
+the MetalLB Operator. The MetalLB Operator is a charm bundle that allows the 
+deployment of both the controller and speaker components.
+
+To deploy the operator, you will first need a Kubernetes model in Juju.
+Add your Kubernetes as a cloud to your Juju controller:
+
+```
+juju add-k8s k8s-cloud --controller $(juju switch | cut -d: -f1)
+```
+
+And create a new Kubernetes model:
+
+```
+juju add-model metallb-system k8s-cloud
+```
+
+Then deploy the MetalLB operator:
+
+```bash
+juju deploy cs:~charmed-kubernetes/metallb-operator
```

Comment:
I may be missing something obvious, but this doesn't seem to work. is that the right namespace?

Reply:
Ah, it's because the main PR is still under review and the charms are not published yet under charmed-kubernetes. It's dependent on this https://github.com/charmed-kubernetes/metallb-operator/pull/1

Reply:
i changed this to the containers namespace as I think that's where it will be. I'm going to merge this and the notes now so I can cue them up for publishing